### ISSUE TITLE
H1 Headings are not included in the automatic Table of Contents on Bloom Docs

### ISSUE THREAD

**Issue by jamespostsil** [sillsdev/docu-notion#104]
**Describe the bug**
When a block is marked as Heading 1, it is not included in the TOC on the right-hand side of the page.

URL of a source Notion page that, when docu-notion accesses or converts it, shows the problem:

Small example: https://www.notion.so/hattonjohn/Is-Bloom-right-for-my-project-58be5d048c7f4fd0aa827dda09c4d345?pvs=4
Large example: https://www.notion.so/hattonjohn/Older-Release-Notes-e24605986bcc4957afb33d02b4cb56e3?pvs=4

**Expected behavior**
H1s should be included in the TOC, or never used in Docu-Notion (not recommended, it would be difficult to remember/implement consistently).

**Screenshots**

![image](https://github.com/sillsdev/docu-notion/assets/106943065/91bf1eda-76aa-4bb9-88c2-a377133a17f1)

Below:
Left-side: The Notion version of the TOC from the Older Release Notes page (I added it temporarily, to compare). Missing H1 items are in orange.
Right-side: The automatic TOC on Bloom Docs.

![image](https://github.com/sillsdev/docu-notion/assets/106943065/87fbc98c-c0d3-4f1b-bc95-9cf002b5e9dc)

**Additional context**

**Comment by andrew-polk** [comment]
See https://docusaurus.io/docs/next/markdown-features/toc#table-of-contents-heading-level for how this can be configured in docusaurus at the page level or for all pages.
(The default is only H2 and H3.)

**Comment by jamespostsil** [comment]
I have made made a mistake, but I tried to implement this on my local version and got an error:

> [ERROR] ValidationError: "tableOfContents.minHeadingLevel" must be greater than or equal to 2

My mod is at the bottom of the themeConfig: section of docusaurus.config.js:

```
themeConfig:
    /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
    ({
      metadata: [
        {
          name: "keywords",
          content: "Bloom, SIL, documentation, help, training",
        },
      ],
      sitemap: {
        // https://www.sitemaps.org/protocol.html#xmlTagDefinitions
        changefreq: "weekly",
        priority: 0.5,
        ignorePatterns: [],
      },
      navbar: {
        title: "Bloom Docs",
        logo: {
          alt: "Bloom Site Logo",
          src: "img/logo.svg",
        },

        items: [
          {
            type: "doc",
            label: "PDF Downloads",
            position: "right",
            docId: "downloads",
          },
          // As of Aug 2023, we don't really have enough translation done to warrant having a locale chooser.
          // {
          //   type: "localeDropdown",
          //   position: "right",
          // },
        ],
      },

      prism: {
        theme: lightCodeTheme,
        darkTheme: darkCodeTheme,
      },

      tableOfContents: {
        minHeadingLevel: 1,
        maxHeadingLevel: 3,
      },
    }),
```

**Comment by hatton** [comment]
Yup, two is the minimum: https://docusaurus.io/docs/api/themes/configuration

**Comment by jamespostsil** [comment]
Hmmm... that's a bit frustrating, because Notion only has H1, H2, and H3...
So for us, the TOC is limited.

Unless docu-notion was set to automatically lower the H-count for pages
with H1 by 1... H1 becomes H2, etc. Then we could fit within docusaurus'
confines (using H2 - H4 instead of H1 - H3).

I realize that would mean more programming. So, what's the priority on
this? Probably not high enough at the moment.
